app/scripts/controllers/necesidad/solicitud_necesidad.js

```diff
@@ -183,36 +184,23 @@ angular
             self.dependencia_destino = response.dependencia.Id;
           }): _;
 
-          self.Necesidad.DependenciaNecesidadId.OrdenadorGastoId
-            ? necesidadService
-                .get_info_dependencia(
-                  self.Necesidad.DependenciaNecesidadId.OrdenadorGastoId
-                )
-                .then(function (response) {
-                  self.rol_ordenador_gasto = response.dependencia.Id;
-                })
-            : _;
-
-          if (self.Necesidad.DependenciaNecesidadId.InterventorId === 0) {
-            //verifica si es supervisor o interventor
+          self.Necesidad.DependenciaNecesidadId.OrdenadorGastoId ? necesidadService.get_info_dependencia(
+            self.Necesidad.DependenciaNecesidadId.OrdenadorGastoId
+          ).then(function (response) {
+            self.rol_ordenador_gasto = response.dependencia.Id;
+          }): _;
```

Comment:
(En general y/o donde aplique) Uso innecesario de ternas, si no se necesita el valor retornado por una terna usar un if-else común...
```suggestion
          if (self.Necesidad.DependenciaNecesidadId.OrdenadorGastoId) {
            necesidadService
            .get_info_dependencia(self.Necesidad.DependenciaNecesidadId.OrdenadorGastoId)
            .then(function (response) {
                self.rol_ordenador_gasto = response.dependencia.Id;
            });
          }
```

Reply:
Se ajustaron algunas lineas con respecto a esto pero realmente son tantas que no podría realizar el ajustes de todas en este momento, pero lo tendré en cuenta para cada caso que vea dentro de los desarrollos.